configure

```diff
@@ -576,11 +576,12 @@ def get_version_helper(cc, regexp):
 
 def get_llvm_version(cc):
   return get_version_helper(
-      cc, r"(^clang version|based on LLVM) ([3-9]\.[0-9]+)")
+    cc, r"(^clang version|^FreeBSD clang version|based on LLVM) " +
```

Comment:
An alternative would be a non-capturing group, e.g. `cc, r"(^(?:FreeBSD )?clang version|based on LLVM) " +`.